java/openmldb-taskmanager/src/main/scala/com/_4paradigm/openmldb/taskmanager/spark/SparkJobManager.scala

```diff
@@ -84,11 +84,7 @@ object SparkJobManager {
     }
 
     // TODO: Avoid using zh_CN to load openmldb jsdk so
```

Comment:
It may be better to remove this comment as well.

Reply:
> It may be better to remove this comment as well.

maybe you help edit the source file, and let's merge this PR after that?

Reply:
We will update the comment later.